odoo/addons/base/models/ir_translation.py

```diff
@@ -426,8 +427,13 @@ def _sync_terms_translations(self, field, records):
                     translations_to_match.append(translation)
 
             for translation in translations_to_match:
-                matches = get_close_matches(translation.src, terms, 1, 0.9)
-                src = matches[0] if matches else None
+                src_text_content = lxml.html.fromstring(translation.src).text_content()
+                terms_text_content = {}
+                for term in terms:
+                    terms_text_content[lxml.html.fromstring(term).text_content()] = term
```

Comment:
move this above to avoid recomputing it for each item of `translations_to_match`

Reply:
Good point.
Code amended.